plugins/sentinel/sentinel/mapping/sentinel_incident.py

```diff
@@ -115,14 +117,17 @@ def get_customer_name(self, tid: str) -> str:
         """
         return self.tid_mapping.get(tid, self.DEFAULT_CUSTOMER_NAME)
 
-    def map_sentinel_status_to_howler(self, sentinel_status: str) -> str:
-        """Map Sentiel Incident status to Howler status.
+    def map_sentinel_status_to_howler(self, sentinel_status: str | None) -> str:
```

Comment:
sadly, until we remove 3.9 support, we have to use Optional:

```suggestion
    def map_sentinel_status_to_howler(self, sentinel_status: Optional[str]) -> str:
```